viewer/core/static/js/app/components/containers/chart-container.js

```diff
@@ -228,6 +228,8 @@ class ChartContainer extends React.Component {
     } else {
       return (
         <Chart
+          {...this.props}
```

Comment:
Nothing that I'm aware of. But I've come to feel it's a best practice. I ran into a couple of bugs during development where something from `{...this.props}` was overriding something I manually wrote and it really tripped me up.

When `{...this.props}` comes first, everything manually defined takes precedence.